app/__init__.py

```diff
@@ -11,6 +12,10 @@
 
 API_VERSION = "1.0"
 
+# Connect to Agolia
+client = SearchClient.create('your_project_name', 'your_key')
```

Comment:
Can we call this `search_client` or `agolia_client` perhaps? In the routes.py file it's not entirely clear what service this is a client of. I'm a fan of `search_client` because if we ever change vendors (perhaps one day we move to elastic search?) then we probably won't have to change the name. And if we ever use another client for something completely different, it won't be confused with that, because we know we're talking about the _search_ client specifically.